data/mods/Magiclysm/Spells/attunements/Cleansing_Flame.json

```
@@ -75,5 +75,62 @@
     "min_range": 4,
     "max_range": 8,
     "range_increment": 0.12
+  },
+  {
+    "id": "cleansingflame_hermes_heal",
+    "type": "SPELL",
+    "spell_class": "CLEANSING_FLAME",
+    "name": "Hermes reborn",
+    "description": "In the sea without lees \nStandeth the bird of Hermes \nEating his wings variable \nAnd maketh himself yet full stable",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
+    "shape": "blast",
+    "base_casting_time": 350,
+    "base_energy_cost": 800,
+    "energy_source": "MANA",
+    "difficulty": 10,
+    "min_damage": -20,
+    "max_damage": -120,
+    "damage_increment": -2.5,
+    "max_level": 40,
+    "extra_effects": [ { "id": "cleansingflame_hermes_burn" }, { "id": "cleansingflame_hermes_split" } ],
+    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE", "NO_PROJECTILE", "IGNORE_WALLS", "LOUD", "MUST_HAVE_CLASS_TO_LEARN" ]
+  },
+  {
+    "id": "cleansingflame_hermes_burn",
+    "type": "SPELL",
+    "name": "Hermes reborn fire",
+    "description": "Give some frost damage for hermes reborn spell. Technical spell, you dont allowed to see it, if youre not in debug mode.",
+    "valid_targets": [ "self", "ally", "hostile", "ground" ],
+    "flags": [ "CONCENTRATE", "SOMATIC", "LOUD", "NO_PROJECTILE" ],
```

```suggestion
    "flags": [ "LOUD", "NO_PROJECTILE" ],
```
N/A for subspells, AFAIK.